driver.go

```diff
@@ -294,7 +325,7 @@ func (l *LinstorDriver) Mount(req *volume.MountRequest) (*volume.MountResponse,
 	if err = l.mounter.MakeDir(target); err != nil {
 		return nil, err
 	}
-	err = l.mounter.FormatAndMount(source, target, params.FS, params.MountOpts)
+	err = l.mounter.Mount(source, target, params.FS, params.MountOpts)
```

Comment:
We should reference the `FileSystem/Type` property here as the mount call receives global config options only.

Reply:
good point. why did that work? :D

Reply:
I saw in linstor-csi the resource options were serialized into an Aux property so it could be read in mount but I wasn't really sure what format to use..